package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/PrimMap.cs

```diff
@@ -48,19 +48,23 @@ public struct InstanceRoot
         public SdfPath[] Skeletons { get; set; }
         public SdfPath[] Materials { get; set; }
 
+        // Flags for import analytics
+        public bool ContainsPointInstances { get; set; }
+
         // Normal objects in the hierarchy.
         private Dictionary<SdfPath, GameObject> m_prims = new Dictionary<SdfPath, GameObject>();
 
         // Objects at the root of an instanced sub-tree.
         // Instances may be found in masters as well.
-        private Dictionary<SdfPath, InstanceRoot> m_instanceRoots = new Dictionary<SdfPath, InstanceRoot>();
+        public Dictionary<SdfPath, InstanceRoot> InstanceRoots = new Dictionary<SdfPath, InstanceRoot>();
```

Comment:
Seems like this change is no longer necessary with the addition of ContainsPointInstances?

Reply:
Yep- forgot to revert it. Thanks :)